src/components/progress/themes/circular/shared/circular.progress.common.scss

```diff
@@ -26,27 +26,22 @@ $theme: $material;
     stroke: var-get($theme, 'base-circle-color');
 }
 
+// Mixin for Gradient Colors
+@mixin gradient-variant($variant, $color) {
+    :host([variant='#{$variant}']) [part~='gradient_start'],
+    :host([variant='#{$variant}']) [part~='gradient_end'] {
+        stop-color: $color;
+    }
+}
```

Comment:
I like that you're optimizing this! Can we instead use a CSS variable to set the step-color for the different variants to reduce the amount of generated CSS?